lib/player.rb

```diff
@@ -1,16 +1,15 @@
 class Player
   attr_reader  :name,
-                    :deck,
-                    :lost
+               :deck
 
   def initialize(name, deck)
     @name = name
     @deck = deck
-    @lost = false
   end
 
   def has_lost?
-    @lost = true if @deck.cards.empty? == true
```

Comment:
this is another spot where we can rely on the fact that:

```ruby
deck.cards.empty?
```
returns `true` and `false` by default. No `if` or variable assignment required!